File: src/IsbInterop/Base/IObjectInfo.vb
Namespace Base
  ''' <summary>
  ''' Информация об объекте.
  ''' </summary>
  Public Interface IObjectInfo
    Inherits IBaseIsbObject
    ''' <summary>
    ''' ИД документа.
    ''' </summary>
    ReadOnly Property Id() As Integer

    ''' <summary>
    ''' Имя документа.
    ''' </summary>
    ReadOnly Property Name() As String
  End Interface
End NameSpace

File: src/IsbInterop/Base/IObjectInfo.cs
namespace IsbInterop.Base
{
  /// <summary>
  /// Информация об объекте.
  /// </summary>
  public interface IObjectInfo : IBaseIsbObject
  {
    /// <summary>
    /// ИД документа.
    /// </summary>
    int Id { get; }

    /// <summary>
    /// Имя документа.
    /// </summary>
    string Name { get; }
  }
}
